frontend/src/store/FamLoginUserState.ts

```diff
@@ -45,6 +46,15 @@ const state = ref({
 
 // --- getters
 
+const getUserType = () => {
```

Comment:
Hi Jota, I wonder maybe we rename get to getUserIdpPovider? I feel overall looks good, but this "usertype" makes me confused with the real user type

Reply:
I do feel confused as well.
From user point of view, it's their user type, but from code it is derived from token claim (the IDP).
Maybe as Catherine suggested, just use "**getUserIdpProvider**".
But would the code below works? (no need to declare extra variable `let userType =`)
```
       // FAM currently supports IDIR/BCeID Business login.
       const idpProvider = state.value.famLoginUser!.idpProvider!.split('-')[1];
	if (IdpProvider.IDIR == idpProvider)
		return IdpProvider.IDIR
	else 
		return IdpProvider.BCEIDBUSINESS
```

Reply:
Hello! I made the name change to "**getUserIdpProvider**", and followed Ian's code suggested as it worked fine.
Thanks!